cloud/migration-stork.md

```diff
@@ -115,14 +115,59 @@ status:
   schedulerStatus: ""
   storageStatus: ""
 ```
-5. Once you apply the above spec you should be able to check the status of the pairing. On a successful pairing, you should 
+5. (EKS Only) When pairing with an EKS cluster, you also need to pass in your
+   AWS credentials which will be used to generate the IAM token. This can be
+   achieved in 2 ways
+   1. Create a secret and mount in the stork deployment (Secure)
```

Comment:
It wasn't clear that this was supposed to happen on the source cluster. Lets try to be overly obvious about that for readers.

Reply:
Done.